eclass/unpacker.eclass

```diff
@@ -356,6 +356,9 @@ _unpacker() {
 	*.lz)
 		: ${UNPACKER_LZIP:=$(type -P plzip || type -P pdlzip || type -P lzip)}
 		comp="${UNPACKER_LZIP} -dc" ;;
+	*.zst)
+		zstd -df "${zst}" -o "${P}.tar"
+		tar -xpf "${P}.tar" -C "${WORKDIR}"
```

Comment:
```suggestion
		comp="zstd -dc"
```

Reply:
Thanks a lot! But I add also the "-f option" (zstd -dfc) because here is a symbolic link. Preparing a new commit/patch...

Reply:
We aren't do that for any of the other decompressors, but it cannot hurt I guess.

Reply:
Okay.

Otherwise (without -f) zstd was giving an error:

>>> Unpacking source...
>>> Unpacking worker-4.6.1.tar.zst to /var/tmp/portage/app-misc/worker-4.6.1/work
Warning : /var/tmp/portage/app-misc/worker-4.6.1/distdir/worker-4.6.1.tar.zst is a symbolic link, ignoring
tar: This does not look like a tar archive
tar: Exiting with failure status due to previous errors
 * ERROR: app-misc/worker-4.6.1::localrepo failed (unpack phase):
 *   unpacking /var/tmp/portage/app-misc/worker-4.6.1/distdir/worker-4.6.1.tar.zst failed (comp=zstd -dc arch=tar --no-same-owner -xof)